packages/assets/scss/03-organisms/_page-header.scss

```diff
@@ -214,6 +214,10 @@ $page-header-widget-width: 350px;
     text-shadow: none;
     margin-top: 0.5rem;
     margin-right: 0.5rem;
+
+    a.ma__button {

```

Comment:
Instead of overriding it here, can we add this as an option to button as a component? For all buttons, if the label text is longer than 2 words, we should not use ALL CAPS. 

This is what I suggest we do:
We move the "text-transform: uppercase;" on ma__button into a modifier class ".ma__button--uppercase". Add a casing prop to buttons, and by default set uppercase to true. Add here we set it to false. 

Also, we should set "$letter-spacing-large;" with the uppercase option, and by with regular casing, we should set it letter spacing to "$letter-spacing-small;"

![Screen Shot 2024-03-11 at 3 32 25 PM](https://github.com/massgov/mayflower/assets/5789411/3b63350a-516e-407d-8aab-bacb71ce1cc2)

Reply:
Changed the implementation as suggested above.